src/applications/dhp-connected-devices/components/ConnectedDevicesContainer.jsx

```diff
@@ -9,8 +9,10 @@ import {
   DISCONNECTION_FAILED_STATUS,
   DISCONNECTION_SUCCESSFUL_STATUS,
 } from '../constants/alerts';
+import { DeviceConnectionAlert } from './DeviceConnectionAlerts';
```

Comment:
See linting suggestion above, can we please update the import ?

Reply:
Hey, I'm not sure what this linting suggestion is

Reply:
Updated the imports for these common modules to use Babel aliases defined in `babel.config.json`